### PR TITLE
IGNITE-15059 Updated Apache Ignite 2.11 release notes

### PR DESCRIPTION
**Pull request by xintrian**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,119 @@
...
+* Fixed NullPointerException when run out automatic segment archiving after last record logging.
+* Fixed possible cluster fail when the socket read hangs during establishing a new communication connection
+* Fixed node not joining baseline topology after auto adjust is enabled
+* Fixed inconsistent WAL mode state when changing the state on unstable topology
+* Fixed Allocated RAM metric for in-memory data regions on persistent node
+* Fixed - safe restart of node with switching from disabled to enabled WAL archiving.
+* Fixed - sync-free switch on node(s) fail implemented. Cellular switch's Multi-cell transaction's recovery HB of node(s) fail fixed.
+* Fixed an issue that led to JVM crash during index removal on a deactivated cluster
+* Fixed bug when messaging-related metrics were updated before full initialization of Tcp Communication SPI.
+* Fixed - partitions validator check sizes even if update counters are different and print information about all copies, if partition is in inconsistent state.
```
Comment (Silberfuchs1) [inline RELEASE_NOTES.txt:66]: checks

Comment (Silberfuchs1) [reply RELEASE_NOTES.txt:66]: prints

File: RELEASE_NOTES.txt [hunk 2]
```diff
@@ -1,6 +1,119 @@
...
+* Fixed - Restart rebuild indexes, if they were not rebuilt, on node restart / reactivation.
+* Extended the metrics log message with information about the striped pool.
+* Implemented asynchronous execution of durable background tasks.
+* Improved log of node ping failure. Now failure reason is logged.
+* Migrated Spring Cache integration to the Ignite extensions
+* Migrated Spring-transactions module to ignite-extensions
+* Optimized collection of rebalance statistic. This has positive influence on rebalance time.
```
Comment (Silberfuchs1) [inline RELEASE_NOTES.txt:73]: statistics

File: RELEASE_NOTES.txt [hunk 3]
```diff
@@ -1,6 +1,119 @@
...
+* Removed of diagnostics at a node stop due to an CorruptedTreeException.
```
Comment (Silberfuchs1) [inline RELEASE_NOTES.txt:74]: you don't need 'of' here.

**Review by alex-plekhanov — APPROVED** [review]

**Pull request closed without merging**